### DISCUSSION TITLE
Is it possible to measure PSNR/SSIM for bicubic upscaling/original data on video dataset

### DISCUSSION THREAD

**Discussion by volotat** [open-mmlab/mmediting#1225] [How-to]
I use mmediting/tools/test.py for measuring metrics for different VSR algorithms and it works as expected. But I also need to establish some baselines PSNR/SSIM values for bicubic upscaling and original data (without any change at all). I wonder if it's possible to use the same script for this task, or any other scripts from mmediting toolkit? And if so how exactly to do it?

**Comment by zengyh1900** [comment]
Please check this issue. @plyfager
